catalog/spatial/csw/spatial-csw-endpoint/pom.xml

```diff
@@ -134,6 +151,7 @@
                             net.opengis.gml.v_3_1_1;version="[2.8,3)",
                             net.opengis.ows.v_1_0_0;version="[2.8,3)",
                             ddf.security.permission,
+                            org.apache.commons.logging; version="[1.1.0,2.0.0)",
```

Comment:
Missed that earlier but we probably should remove `org.apache.commons.logging` as well.